### ISSUE TITLE
SlevomatCodingStandard.TypeHints.NullableTypeForNullDefaultValue not working

### ISSUE THREAD

**Issue by PostScripton** [slevomat/coding-standard#1350]
`SlevomatCodingStandard.TypeHints.NullableTypeForNullDefaultValue` rule doesn't wokr at all.

```xml
<rule ref="SlevomatCodingStandard.TypeHints.NullableTypeForNullDefaultValue"/>
```

```php
public function someMethod(?int $foo, int $bar = null): array
{
}
```

None of them are working, a ruleset is simply passes.

**Comment by kukulich** [comment]
I've just tested it and it works as expected, even for your code.

So there's probably something wrong in your configuration. Please try to use verbose mode of PHPCS to check it.

**Comment by PostScripton** [comment]
I'm so sorry, Docker didn't updated my ruleset configuration file. It works as expected after restart
